apps/web/src/components/activity/ExecutionDetailsSteps.tsx

```diff
@@ -0,0 +1,287 @@
+import { Grid, Text } from '@mantine/core';
+import styled from 'styled-components';
+import { format, isValid, parseISO } from 'date-fns';
+import { ExecutionDetailsStatusEnum } from '@novu/shared';
+
+import { colors, shadows } from '../../design-system';
+import { Chat, CheckCircle, InApp, Mail, Sms, Timer } from '../../design-system/icons';
+
+const ExecutionDetailsStepItem = styled.div`
```

Comment:
I would suggest moving all styles to the bottom of the page, after the component, so opening the file you can immediately understand what its about